internal/ingress/annotations/annotations.go

```diff
@@ -111,35 +107,22 @@ func IngressClassValidatorFuncFromV1Ingress(
 	}
 }
 
-// valueFromAnnotation returns the value of an annotation with key.
-// key is without the annotation prefix of configuration.konghq.com or
-// konghq.com.
-// It first looks up key under the konghq.com group and if one doesn't
-// exist then it looks up the configuration.konghq.com  annotation group.
+// valueFromAnnotation returns the value of an annotation with key konghq.com.
 func valueFromAnnotation(key string, anns map[string]string) string {
-	value, exists := anns[annotationPrefix+key]
-	if exists {
-		return value
-	}
-	return anns[deprecatedAnnotationPrefix+key]
+	return anns[AnnotationPrefix+key]
```

Comment:
There is no value left in this function. The caller can do `value = anns[AnnotationPrefix+key]` at this point.